modules/aggregation-layers/src/common/utils/bounds-utils.ts

```diff
@@ -0,0 +1,30 @@
+/** Utility to estimate binIdRange as expected by AggregatorProps */
+export function getBinIdRange({
```

Comment:
Why is it only an estimate?
```suggestion
export function estimateBinIdRange({
```